Documentation/Storage-Configuration/Block-Storage-RBD/block-storage.md

```diff
@@ -197,3 +199,32 @@ The erasure coded pool must be set as the `dataPool` parameter in
 The erasure coded pool must be set as the `dataBlockPool` parameter in
 [`storageclass-ec.yaml`](https://github.com/rook/rook/blob/master/deploy/examples/flex/storage-class-ec.yaml). It is used for
 the data of the RBD images.
+
+### Recover RBD RWO volume in case of Node Loss
+
+!!! Note
+    The minimum required Kubernetes version require for this to support is v1.26 and assuming [CSI-add-ons](https://rook.github.io/docs/rook/latest/Storage-Configuration/Ceph-CSI/ceph-csi-drivers/?h=csiaddons#csi-addons-controller) CRD is deployed.
+
+In case, a node is lost where a pod is running with the RBD RWO volume is mounted, the volume cannot automatically be mounted on another node. To recover this situation, add the below taint on the node once admin confirms node is down.
```

Comment:
Let's move this intro paragraph before the note.
```suggestion
In case, a node is lost where a pod is running with the RBD RWO volume is mounted, the volume cannot automatically be mounted on another node. To recover, confirm the node is down and then add the below taint to the node
```